daemon/cluster/convert/swarm.go

```diff
@@ -12,6 +13,14 @@ import (
 
 // SwarmFromGRPC converts a grpc Cluster to a Swarm.
 func SwarmFromGRPC(c swarmapi.Cluster) types.Swarm {
+	var defaultAddrPool []*net.IPNet
+	for _, address := range c.DefaultAddressPool {
+		_, b, err := net.ParseCIDR(address)
```

Comment:
nit; given that `b` is only used here, this could be;

```go
if _, b, err := net.ParseCIDR(address); err == nil {
    defaultAddrPool = append(defaultAddrPool, b)
}
```

Reply:
We are adding into slice only if b is not null. Its not likely to happen hence I will keep the logic as it is.

Reply:
> We are adding into slice only if b is not null.

Wrong, you're adding it to the slice only if `err == nil`, there is no check on `b`. I do agree with @thaJeztah and the proposed change :angel: